packages/libraries/core/src/client/types.ts

```diff
@@ -73,6 +73,12 @@ export interface Logger {
 }
 
 export interface HiveUsagePluginOptions {
+  /**
+   * The target to which the usage data should be reported to.
+   * This can either be a slug following the format "$organizationSlug/$projectSlug/$targetSlug" (e.g "the-guild/graphql-hive/staging")
+   * or an UUID (e.g. "a0f4c605-6541-4350-8cfe-b31f21a4bf80").
```

Comment:
```suggestion
   * This can either be a slug following the format `$organizationSlug/$projectSlug/$targetSlug` (e.g `the-guild/graphql-hive/staging`)
   * or an UUID (e.g. `a0f4c605-6541-4350-8cfe-b31f21a4bf80`).
```